spectral.yml

```
@@ -124,7 +120,7 @@ rules:
       functionOptions:
         notMatch: "\/(get|post|put|patch|delete)(\/|$)"
 
-  ## checks that every route has a description, the description of every http verb is mandatory by spectral already ##
+  ## checks that every route has a description, and the description of every http verb is mandatory by spectral already ##
```

Thanks for your contribution :)
Just this change bothers me a bit, since it alters the meaning of the comment. The second part after the comma is intended to tell everybody that a rule for the individual HTTP verbs already exists. Reasoning why it was only implemented for the general route. But I agree that the previous version was misleading as well. How about we split this into two sentences, would that be alright for you?
```suggestion
  ## checks that every route has a description. The description of every http verb is mandatory by spectral already ##
```

---

like @markbrockhoff approach 👍